Codeinwp/neve
# fix: global colors palette selector in safari [closes #2479]

### Summary
Fixes issue where global colors palette selector color swatches were not visible in some Safari versions.


### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Test inside Browserstack with Safari 13.1 on macOS Catalina 
- You can just launch a jurassic.ninja instance with the version from WP.org
- Check that the color swatches inside Colors & Background in Customizer are not showing up
- Upload the zip from this PR and replace the theme
- The color swatches should be working as expected


Closes #2479.
